### PR TITLE
OADP-3268 [DOC] Release notes small corrections

### PR DESCRIPTION
**Pull request by anarnold97**
### Jira 

* [OADP-3268](https://issues.redhat.com/browse/OADP-3268)

### Versions:

* OCP 4.12 → branch/enterprise-4.12
* OCP 4.13 → branch/enterprise-4.13
* OCP 4.14 → branch/enterprise-4.14
* OCP 4.15 → branch/enterprise-4.15

### Docs preview:

* [Docs preview OADP 1.3 release notes](https://69552--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/release-notes/oadp-release-notes-1-3)
 * [Docs preview OADP 1.2 release notes](https://69552--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/release-notes/oadp-release-notes-1-2)
 * [Docs preview OADP 1.1 release notes](https://69552--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/release-notes/oadp-release-notes-1-1)

### CODE REVIEW

**Comment by anarnold97** [comment]
@mburke5678 - thanks big man

File: modules/oadp-release-notes-1-3-0.adoc [hunk 1]
```diff
@@ -149,7 +149,7 @@ spec:
 
 [NOTE]
 ====
-In OADP 1.4, the `kopia` will become the default `uploaderType` value.
+In OADP 1.4, `kopia` will become the default `uploaderType` value.
```
Comment (michaelryanpeter) [inline modules/oadp-release-notes-1-3-0.adoc:152]: Not a merge blocker, but please consider creating a follow up PR to reword this sentence. The [ISG states](https://www.ibm.com/docs/en/ibm-style?topic=information-claims-recommendations) that we should avoid claims about future product releases. 

```suggestion
In a future OADP release, it is planned that the `kopia` tool will become the default `uploaderType` value.
```

Comment (michaelryanpeter) [reply modules/oadp-release-notes-1-3-0.adoc:152]: This is the format that PX has asked us to use when talking about plans for future releases.

**Pull request merged**